lib/action_subscriber/configuration.rb

```diff
@@ -79,6 +88,13 @@ def add_decoder(decoders)
       self.decoder.merge!(decoders)
     end
 
+    def connection_string=(url)
+      settings = ::AMQ::Settings.parse_amqp_url(url)
+      settings.each do |key, value|
+        send("#{key}=", value)
```

Comment:
if we use the full names then this probably won't work huh? Because `::AMQ::Settings` will try to pass the `key` as `vhost`, `pass`, etc...